src/pybind/mgr/k8sevents/module.py

```diff
@@ -1053,27 +1168,145 @@ def show_events(self, events):
             return 0, "", "No events emitted yet, local cache is empty"
 
     def show_status(self):
-        s = "Tracker Health\n"
+        s = "Kubernetes\n"
+        s += "- Hostname  : {}\n".format(self.k8s_config['server'])
+        s += "- Namespace : {}\n".format(self._namespace)
+        s += "Tracker Health\n"
         for t in self.trackers:
-            s += "- {} : {}\n".format(t.__class__.__name__, t.health)
+            s += "- {:<20} : {}\n".format(t.__class__.__name__, t.health)
         s += "Tracked Events\n"
-        s += "- namespace  : {:>3}\n".format(len(self.ns_watcher.events))
-        s += "- ceph events: {:>3}\n".format(len(self.event_processor.events))
+        s += "- namespace   : {:>3}\n".format(len(self.ns_watcher.events))
+        s += "- ceph events : {:>3}\n".format(len(self.event_processor.events))
         return 0, "", s
 
+    def _valid_server(self, server):
+        # must be a valid server url format
+        server = server.strip()
+
+        res = urlparse(server)
+        port = res.netloc.split(":")[-1]
+
+        if res.scheme != 'https':
+            return False, "Server URL must use https"
+
+        elif not res.hostname:
+            return False, "Invalid server URL format"
+
+        elif res.hostname:
+            socket.setdefaulttimeout = 1
```

Comment:
Do you need to reset the timeout again?
```python
old_timeout = socket.getdefaulttimeout()
try:
    socket.setdefaulttimeout = 1
    try:
        socket.gethostbyname(res.hostname)
    except socket.gaierror:
        return False, "Unresolvable server URL"
finally:
    socket.setdefaulttimeout = old_timeout
```

Reply:
no - it's a default :) But it should be set earlier not at each call! thanks

Reply:
no, a function :-) https://docs.python.org/2/library/socket.html#socket.setdefaulttimeout